admin/sql/updates/20160604-mbs-8951.sql

```diff
@@ -0,0 +1,4 @@
+\set ON_ERROR_STOP 1
+
+DROP INDEX CONCURRENTLY edit_note_idx_post_time_edit;
+CREATE INDEX CONCURRENTLY edit_note_idx_post_time_edit ON edit_note (post_time DESC NULLS LAST, edit DESC);
```

Comment:
Yeah, we should perhaps make a ticket to collect these things. There are more changes in #311 that need to be carried over, too, and more index changes might be done before the next schema change (whenever it will be).